azure-templates/stages.yml

```diff
@@ -82,19 +82,13 @@ stages:
           - name: diffVersion
             value: $[ dependencies.Find64bitVersion.outputs['FindLast64bitVersion.version'] ]
         steps:
-          - template: steps-pre-build.yml # Configure variables, check out repos, Clear cache
+          - template: steps-prepare.yml # Configure variables, check out repos, Clear cache
             parameters:
-              lvVersionToBuild:
-                version:               $(diffVersion)
-                bitness:               $(diffBitness)
-              releaseVersion:          ${{ parameters.releaseVersion }}
-              archiveLocation:         ${{ parameters.archiveLocation }}
-              buildOutputLocation:     ${{ parameters.buildOutputLocation }}
-              codegenVis:              ${{ parameters.codegenVis }}
-              submodules:              ${{ parameters.submodules }}
+              build: False
+              clearCache: False
           - template: steps-diff.yml
             parameters:
-              lvVersionToDiff:         $(diffVersion)
+              lvVersionToDiff: $(diffVersion)
```

Comment:
Now that we aren't passing the LV version and bitness to the pre-build step, we can simplify the diff stage a bit. Can remove lines 68, 70, and 80-81 because diffBitness is no longer used (and could have been hard-coded as 64-bit originally if I'd have thought about it).